pong_game.js

```diff
@@ -1,8 +1,8 @@
 
```

Comment:
this file's changes are mostly redundant for the scope of current state of the project